Neos.Fusion/Classes/Core/Runtime.php

```diff
@@ -103,6 +107,8 @@ class Runtime
      */
     public readonly FusionGlobals $fusionGlobals;
 
+    public readonly HttpResponseConstraints $unsafeHttpResponseConstrains;
```

Comment:
todo add comment that this, like before with response manipulation of the controller context, will only work consistently for uncached segments. For other cases it might only be used once for the first uncached request.